code/game/machinery/ai_slipper.dm

```diff
@@ -4,133 +4,66 @@
 	icon_state = "liquid_dispenser"
 	layer = 3
 	plane = FLOOR_PLANE
-	anchored = 1.0
+	anchored = TRUE
 	max_integrity = 200
 	armor = list(melee = 50, bullet = 20, laser = 20, energy = 20, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 30)
 	var/uses = 20
-	var/disabled = TRUE
-	var/lethal = 0
-	var/locked = TRUE
-	var/cooldown_time = 0
-	var/cooldown_timeleft = 0
+	var/cooldown_time = 10 SECONDS
 	var/cooldown_on = FALSE
 	req_access = list(ACCESS_AI_UPLOAD)
 
+/obj/machinery/ai_slipper/examine(mob/user)
+	. = ..()
+	. += "A small counter shows it has: <B><U>[uses]</U></B> uses remaining."
+
 /obj/machinery/ai_slipper/power_change()
-	if(stat & BROKEN)
-		update_icon()
+	if(powered())
+		stat &= ~NOPOWER
 	else
-		if( powered() )
-			stat &= ~NOPOWER
-		else
-			stat |= NOPOWER
-			disabled = TRUE
-		update_icon()
+		stat |= NOPOWER
+	update_icon()
 
-/obj/machinery/ai_slipper/proc/setState(enabled, uses)
-	disabled = disabled
-	uses = uses
-	power_change()
+/obj/machinery/ai_slipper/attack_ai(mob/user)
+	add_hiddenprint(user)
+	return attack_hand(user)
 
-/obj/machinery/ai_slipper/attackby(obj/item/W, mob/user, params)
+/obj/machinery/ai_slipper/attack_ghost(mob/user)
+	if(user.can_advanced_admin_interact())
+		Activate(user)
+
+/obj/machinery/ai_slipper/attack_hand(mob/user)
 	if(stat & (NOPOWER|BROKEN))
+		to_chat(user, "[src] has no power or is broken!")
```

Comment:
Wrap these and the ones below in the relevant spans (either `warning` or `notice`)